__tests__/api.test.js

```diff
@@ -64,7 +64,45 @@ describe("/api/articles", () => {
       .get("/api/articles")
       .expect(200)
       .then((res) => {
-        expect(res.body.articles).toBeSorted({ key: "created_at",  descending: true});
+        expect(res.body.articles).toBeSorted({
+          key: "created_at",
+          descending: true,
+        });
+      });
+  });
+});
+
+describe("/api/articles/:article_id", () => {
+  test("GET:200. Sends a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article).toMatchObject({
+          article_id: expect.any(Number),
```

Comment:
You can assert that article id will be one here, because it is fixed in your endpoint

Reply:
Done